src/SonarQube.TeamBuild.Integration.Legacy/app.config

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is copied from `SonarQube.TeamBuild.Integration/app.config`. Not sure if we need it...

Reply:
1/ Some of the bindings redirect don't match the version of the NuGet you are referencing.
2/ This is easy to test whether this works or not without the `app.config`.